sandbox.html

```diff
@@ -37,6 +37,18 @@
     kd.run(function () {
       kd.tick();
     });
+    function tick(){
+      if(kd.P.justPressed()){
+        log('The P key was pressed!');
+
+      }
+      if(kd.D.justPressed()){
+        log('The D key was pressed!');
+
+      }
+      requestAnimationFrame(tick)
+    }
+    requestAnimationFrame(tick)
```

Comment:
This seems like a good opportunity to use a Keydrown API and simplify things a bit:

```suggestion    kd.run(function () {
      if (kd.P.wasDownInPreviousTick()) {
        log('The P key was pressed!');
      }
      
      if (kd.D.wasDownInPreviousTick()) {
        log('The D key was pressed!');
      }
    })
```